publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dto/PublicationStatus.java

```diff
@@ -0,0 +1,22 @@
+package no.unit.nva.publication.doi.dto;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum PublicationStatus {
```

Comment:
I imagine this Publication status probably are missing the ARCHIVED one, where ARCHIVED are the publication has a DOI, but it has removed its metadata and now redirects to a tombstone landing page. But we probably don't have to deal with these in the first iteration, and can easily be added later when there is user story about it.

Reply:
Discussed this with @brinxmat and PublicationStatus.REJECTED is wrong and should be named DELETED. I createrd a bug for this: https://unit.atlassian.net/browse/NP-1822

Reply:
@brinxmat But is it really deleted? I believe this is rather archived, as we still need to keep the information about the publication to be able to present a landing page which qualifies the tombstone landing page requirements for the DOI **if it has one**. 

Could argue you have both DELETED and ARCHIVED, where DELETED is fine if it never had any DOI?

Reply:
Regarding above comment @brinxmat , I'm happy we deal with it in the created issue.. out of scope for now so @janbj can continue working.

Reply:
Deleted is the terminology used conversationally — and we don't have a task for this. The semantics for all the terms are unspecified as we don't define semantics for these terms. This is a mistake which we have to rectify.

Reply:
@brinxmat "Archived, everything in an archive is archived" is an eye opener. :+1: